src/utils/getUserBy.js

```diff
@@ -0,0 +1,5 @@
+import usersFromServer from '../api/users';
+const preparedUsers = [...usersFromServer];
+
+export const getUserBy = todo =>
+  preparedUsers.find(user => user.id === todo.userId);
```

Comment:
Consider handling the case where a user might not be found. Returning `null` or a default value can prevent potential runtime errors when the function's result is used.